src/main/java/com/github/seratch/jslack/api/model/dialog/Option.java

```diff
@@ -0,0 +1,15 @@
+package com.github.seratch.jslack.api.model.dialog;
+
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * A dialog element option used with {@link DialogSelectElement}s
+ *
+ */
+@Data
+@Builder
+public class Option {
```

Comment:
We already have a generic `Option` class. https://github.com/seratch/jslack/blob/master/src/main/java/com/github/seratch/jslack/api/model/Option.java

`DialogOption` is preferred for me. Is it fine with you to rename this?

Also, I would like to have `Dialog` prefix in all classes under `com.github.seratch.jslack.api.model.dialog` for consistent naming (= Element -> DialogElement, SubType -> DialogSubType).

Reply:
Yes- that will be clearer. I'll rename the classes